packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -294,21 +297,6 @@ export default function MainFeedLayout({
     };
 
     const variables = getVariables();
-    const mobileExploreActions = !isLaptop &&
-      (isExplorePopular || isExploreLatest) && (
-        <Dropdown
-          className={{ container: 'mx-4 mb-2 !w-56' }}
-          selectedIndex={selectedAlgo}
-          options={algorithmsList}
-          onChange={(_, index) => setSelectedAlgo(index)}
-        />
-      );
-    const controlFeedActions = feedWithActions && (
-      <SearchControlHeader
-        algoState={[selectedAlgo, setSelectedAlgo]}
-        feedName={feedName}
-      />
-    );
```

Comment:
Had to move as the dropdown goes missing once the feed is empty, making the user unable to switch back.